autogluon/inference/docker/0.4/py3/cu112/Dockerfile.gpu

```diff
@@ -25,6 +25,9 @@ RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.p
 # Resolving sagemaker 2.92.1 conflict
 RUN pip install --no-cache-dir 'importlib-metadata<2.0,>=1.4.0' markdown
 
+# Removing GluonTS poetry.lock - creates false alarms on scans
+RUN rm -rf /usr/local/lib/python3.8/dist-packages/gluonts/nursery/tsbench/poetry.lock
```

Comment:
I had a second thought regarding the removal of this package. Is it possible that the customers could be relying on this for their usage?

Reply:
I am apprehensive that we might end up removing something that might cause failures for the customers. Had it been the very first release of a new AG version, this would not have been a problem.

Reply:
tsbench is a benchmark tooling (not sure why it's bundled within the library). We don't expect any production users to use it as a part of normal use of AG.
I did some sanity tests for time series: it does not affect training/inference functionality.

Reply:
Got it, thanks!!